ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -9,22 +9,26 @@
  */
 package org.openmrs.reference;
 
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.HomePage;
 
 import static org.junit.Assert.assertTrue;
 
-@Ignore
-public class LoginTest extends ReferenceApplicationTestBase {
+public class LoginTest extends ReferenceApplicationTestBase {   
+    
```

Comment:
this is trailing space and should be excluded

Reply:
Resolved thanks